.gitignore

```
@@ -1 +1,2 @@
 .ipynb_checkpoints
+data/
```

```suggestion
0.data-download/data/*

```

To be more specific about which data folder. We might want to include other data in other folders in the future.

Also, I'm adding an extra new line character to the end of the file. This removes the red warning label at the end of line two.

p.s. - See [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) for instructions on how to accept "suggestions" into your  PR.